### DISCUSSION TITLE
Order confirmation does not show Custom Order Fields

### DISCUSSION THREAD

**Discussion by leonkosterss** [swellstores/community#541] [Help]
Hello,

When I create an Order in the Dashboard, and add content to the custom order field it will send an Order confirmation mail with those fields.

But when I create an Order by my custom checkout, it sends the Order confirmation without those fields. It fills the fields in the Order page in the Dashboard.

How is this possible? Do I have to create the order in another way?

The domain is: new.obimex.nl -- You can create an Order if you want to test it.

Thanks in advance!

**Comment by sags95** [comment]
How are you setting those fields in the custom checkout, are you directly creating orders with Backend API or using Frontend API?
